### DISCUSSION TITLE
Context problem in custom keybindings

### DISCUSSION THREAD

**Discussion by MinottiAlessandro** [zed-industries/zed#7797] [Feature Requests]
Hi everyone,

I'm experimenting with keybindings and I'm having a problem excluding certain keybindings from a particular context.
More specifically I need to exclude the "Global" keybindings from the "ProjectSearch" context, does anyone of you have an idea on how it could be done?

This is my keybindings file:
```
[
  // ProjectSearch specific
  {
    "context": "ProjectSearchBar",
    "bindings": {
      "escape": "pane::CloseActiveItem"
    }
  },
  // Global
  {
    "context": "(Editor || ProjectPanel) && vim_mode != insert && vim_mode != visual && !VimWaiting",
    "bindings": {
      "space p": "file_finder::Toggle",
      "space f": "pane::DeploySearch",
      "space space": ["buffer_search::Deploy", { "focus": true }],
      "space w": "pane::CloseActiveItem"
    }
  }
]
```

Thank you

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
